portfolio/src/main/java/com/google/sps/servlets/DeleteServlet.java

```diff
@@ -0,0 +1,38 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.FetchOptions;
+import com.google.appengine.api.datastore.FetchOptions.Builder;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/delete-data")
+public class DeleteServlet extends HttpServlet {
+    private final DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
```

Comment:
Don't forget to deal with formatting eventually, both here and in your javascript/html files. Check if your editor of choice can auto indent to two spaces.

Reply:
opened a separate pr for this! will merge it soon.

Reply:
since this is a new class, i would address it here. mind running the formatter for this file?

Reply:
i was having trouble with the formatter, but i think i've fixed it!